### PR TITLE
make prometheus ingress enable work with uncommenting the hosts

### PR DESCRIPTION
**Pull request by LillyWu**

### CODE REVIEW

File: install/kubernetes/helm/istio/values.yaml [hunk 1]
```diff
@@ -296,8 +296,8 @@ prometheus:
   ingress:
```
Comment (douglas-reid) [inline install/kubernetes/helm/istio/values.yaml]: I _think_ the plan is to disable ingress specs, in favor of ingressgateway. 

cc: @costinm

Comment (LillyWu) [reply install/kubernetes/helm/istio/values.yaml]: @douglas-reid  thanks for your comment. If this is the plan, then `ingress` item needs to be removed and also update this installation options to make all clear: https://github.com/istio/istio.github.io/blob/master/content/docs/reference/config/installation-options/index.md

Comment (douglas-reid) [reply install/kubernetes/helm/istio/values.yaml]: @LillyWu agreed. I'm awaiting the dust to settle on ingressgateway refactoring.

**Comment by LillyWu** [comment]
closing this PR as it's needed any more

**Pull request closed without merging**